WordPress/src/main/java/org/wordpress/android/ui/accounts/login/LoginEpilogueFragment.java

```diff
@@ -166,6 +166,34 @@ private SitePickerAdapter getAdapter() {
             mUsernameTextView = (TextView) view.findViewById(R.id.username);
             mMySitesHeadingTextView = (TextView) view.findViewById(R.id.my_sites_heading);
         }
+
+        public void update(Context context, HeaderViewHolder holder, boolean showHeader, AccountModel defaultAccount) {
+            if (showHeader) {
```

Comment:
The mMySitesHeadingTextView needs to be part of the ViewHolder/header in order to scroll along with the 'logged in as' section.

Creating a new ViewHolder and a layout for the 'logged in as' section and second layout which includes 'logged in as' section and adds mMySitesHeadingTextView seems like a huge overkill.

I could wrap 'logged in as' section into a layout and show/hide just the layout. It would save 2 lines in the java code, but add a needless layout to the xml.

Am I missing something?:) 

I divided the showHeader parameter into hasAccessToken and isAfterLogin parameters. The naming was very confusing. Nevertheless, I'll be happy to improve the code more if you clarify your thoughts. Thanks @hypest!

Reply:
> in order to scroll along with the 'logged in as' section.

Ah, I see. Yes, makes sense to keep mMySitesHeadingTextView in the header layout. It doesn't worth the hassle to extract it so, 👍 

> I divided the showHeader parameter into hasAccessToken and isAfterLogin parameters. 

I think `isAfterLogin` is quite clear but `hasAccessToken` not so much. How about renaming it to `isLoggedInWpcom`?